guide/payments/coin-selection.md

```diff
@@ -0,0 +1,312 @@
+   layout = "full-width"
+%}
+
+# Coin selection
+
+Coin selection is the process of choosing which [UTXOs](https://hackmd.io/glossary#utxo) (or “coins”) to fund a bitcoin transaction with (inputs[^1]) when making an on-chain payment. Since your wallet contains multiple coins to arrive at a balance, these are retrieved by an algorithm to fulfill the payment amount for outgoing transactions.
+
+Coin selection, whether an automated or manual process, is important when understanding how we can successfully optimise our bitcoin payments for either cost, speed, or privacy.
+
+There are two categories of coin selection strategies that are used in bitcoin applications:
+
+1. **Automatic Coin Selection** (wallet is delegated to control coin selection on behalf on user)
+2. **Manual Coin Selection** (user controls coin selection)
+
+{% include picture.html
+   image = "/assets/images/guide/glossary/coin-selection/funding-tx.jpg"
+   retina = "/assets/images/guide/glossary/coin-selection/funding-tx@2x.jpg"
+   mobile = "/assets/images/guide/glossary/coin-selection/funding-tx-mobile.jpg"
+   mobileRetina = "/assets/images/guide/glossary/coin-selection/funding-tx-mobile@2x.jpg"
+   alt-text = "Funding TX"
+   width = 800
+   height = 400
+%}
+
+### Hard contraints
+
+Before exploring some popular coin selection strategies, it’s worth noting the following when making a bitcoin payment in relation to UTXOs. 
+
+Firtstly; the transaction must have sufficient funding. For example, in order to send X, the transaction inputs must at least equate to X. Second; as blockspace is limited, miners prioritise transactions by *fee per byte* to maximise revenue. Transactions must be accompanied by the minimum relay fee for confirmation (therefore inputs must equal X + fee). Third; transaction outputs must have a value of 500 sats or more. Anything less than this value is considered  a "dust" output, and these transactions are often not relayed and confirmed by most nodes and miners in the network. Finally; it is essential for wallet owners to label their incoming and outgoing (change) coins. Labels provide context to a wallet’s transaction history and coin provenance, making it easier to successfully employ coin control for privacy optimisation.
+
+### Optimisations
+
+Coin selection will always lead to certain effects and outcomes which can either optimise or hinder our bitcoin payments. Below is a breakdown of the three most common effects of coin control.
+
+#### Cost
+
+Miners select transactions on the basis of their fee, measured by the fee rate of satoshis per byte. Therefore, it is cost effective for users to minimize transaction size by minimising the number of inputs.
+
+#### Speed
+
+Higher fees will increase the likelihood of your transaction being prioritised by miners and included in the next block, thereby increasing the speed of your transaction. This can be optimised by either increasing the fee rate, or by selecting a higher number of transaction inputs.
```

Comment:
```suggestion
Higher fees will increase the likelihood of your transaction being prioritized by miners and included in the next block, thereby increasing the speed of your transaction. This can be optimized by either increasing the fee rate or selecting a higher number of transaction inputs.
```